contracts/fixed_price.scilla

```diff
@@ -1394,6 +1439,15 @@ transition SetAllowlist(
   event e
 end
 
+(* @multi-sig *)
+transition SetOperator(to: ByStr20)
```

Comment:
@razill can you add a transition to clear the operator?

Reply:
@inamirandaz Ideally, we can reuse the same function, I have added a new function to clear Operator